src/Bicep.Core.Samples/ExamplesTests.cs

```diff
@@ -68,6 +68,32 @@ private static IEnumerable<object[]> GetExampleData()
             }
         }
 
+        private static bool IsPermittedMissingTypeDiagnostic(Diagnostic diagnostic)
+        {
+            if (diagnostic.Level != DiagnosticLevel.Warning)
```

Comment:
[Not for this PR] So technically this also would exclude info level in addition to errors. I think it's fine to be this strict in test code to force people to look, but it reminded me of something else.Should we assign numeric values in order of severity to `DiagnosticLevel`, so relational operators can be used with values of that type?

Reply:
Yup, we'll want to do that at some point. I'm not totally sure why I'm even bothering to check for warning here when I'm already looking for a specific code, so I think I'll just remove this check to make the future refactor easier.